contracts/interfaces/IPool.sol

```diff
@@ -571,6 +571,13 @@ interface IPool {
 
   /**
    * @notice Returns the normalized variable debt per unit of asset
+   * IMPORTANT the utility of this function is primarily for the protocol itself,
+   * to get a "dynamic" variable index based on time, current stored index and
+   * virtual rate at the current moment (approx. a borrower would get if opening a position).
+   * This means that is always used in combination with variable debt supply/balances.
+   * If using this function externally, it is fundamental to understand that is possible
+   * to have an increasing normalized variable debt that is not equivalent on how the
+   * variable debt index would be updated in storage (e.g. if there is variable debt supply)
```

Comment:
```suggestion
   * @dev WARNING: This function is intended to be used primarily by the protocol itself, to give an
   * estimate on the variable rate a borrower would get if opening a position. It calculates a "dynamic"
   * variable index based on time, ongoing stored index and current virtual rate.
   * Internally, it is always used in combination with variable debt supply/balances. Consider that is
   * possible to have an increasing normalized variable debt that is not equivalent to how the variable
   * debt index would be updated in storage (e.g. only updates with non-zero variable debt supply)
```

Reply:
Not sure. This is not really precise `This function is intended to be used primarily by the protocol itself, to give an estimate on the variable rate a borrower would get if opening a position`.

```suggestion
   * WARNING: This function is intended to be used primarily by the protocol itself
   * to get a "dynamic" variable index based on time, current stored index and
   * virtual rate at the current moment (approx. a borrower would get if opening a position).
   * This means that is always used in combination with variable debt supply/balances.
   * If using this function externally, Consider that is
   * possible to have an increasing normalized variable debt that is not equivalent to how the variable
   * debt index would be updated in storage (e.g. only updates with non-zero variable debt supply)
```

Reply:
It is fine to me. Consider making each line longer and using a @dev tag